xds/src/main/java/io/grpc/xds/SharedXdsClientPoolProvider.java

```diff
@@ -111,19 +113,19 @@ public ImmutableList<String> getTargets() {
     return ImmutableList.copyOf(targetToXdsClientMap.keySet());
   }
 
-
   private static class SharedXdsClientPoolProviderHolder {
     private static final SharedXdsClientPoolProvider instance = new SharedXdsClientPoolProvider();
   }
 
   @ThreadSafe
   @VisibleForTesting
-  static class RefCountedXdsClientObjectPool implements ObjectPool<XdsClient> {
+  protected class RefCountedXdsClientObjectPool implements ObjectPool<XdsClient> {
 
-    private static final ExponentialBackoffPolicy.Provider BACKOFF_POLICY_PROVIDER =
+    private final ExponentialBackoffPolicy.Provider backoffPolicyProvider =
```

Comment:
This isn't providing any value as `non-static`. Either we should move it up to the provider so it can remain `static` or construct the object when creating XdsClientImpl.

(Actually, we could make Provider a singleton in ExponentialBackoffPolicy itself. But the object is small, so it doesn't matter much.)

Reply:
Moved it up to the provider.